packages/analytics-js-plugins/src/iubendaConsentManager/index.ts

```diff
@@ -20,49 +24,38 @@ const IubendaConsentManager = (): ExtensionPlugin => ({
   },
   consentManager: {
     init(state: ApplicationState, logger?: ILogger): void {
-      // Nothing to initialize
+      // getIubendaUserConsentedPurposes returns current iubenda opted-in purposes
+      // This will be helpful for debugging
+      (globalThis as any).getIubendaUserConsentedPurposes = () =>
+        (state.consents.data.value.allowedConsentIds as string[])?.slice();
+
+      // getIubendaUserDeniedPurposes returns current Iubenda opted-out purposes
+      // This will be helpful for debugging
+      (globalThis as any).getIubendaUserDeniedPurposes = () =>
+        (state.consents.data.value.deniedConsentIds as string[])?.slice();
+
+      // updateKetchConsent callback function to update current consent purpose state
+      // this will be called from ketch rudderstack plugin
+      (globalThis as any).updateIubendaConsent = (iubendaConsentData: IubendaConsentData) => {
```

Comment:
Does Iubenda SDK invoke this callback?

Reply:
No, I am using it in the unit tests only